gateway/api/management/commands/update_jobs_statuses.py

```diff
@@ -56,15 +57,18 @@ def update_job_status(job: Job):
             job.env_vars = "{}"
 
     if job_handler:
+        # let's store logs only for Functions from providers
         logs = job_handler.logs(job.ray_job_id)
-        job.logs = check_logs(logs, job)
-        # check if job is resource constrained
-        no_resources_log = "No available node types can fulfill resource request"
-        if no_resources_log in job.logs:
-            job_new_status = fail_job_insufficient_resources(job)
-            job.status = job_new_status
-            # cleanup env vars
-            job.env_vars = "{}"
+        logs_size_mgb = sys.getsizeof(logs) / (1024**2)
+        if logs_size_mgb < 50:
```

Comment:
I think I would rather truncate the logs than not save them at all, just an opinion. In any case, if we followed this approach, I would add a log saying something like "Logs exceeded maximum allowed size (50 MB) and could not be stored."

Reply:
I agree, I prefer to truncate them